tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Languages/PythonLanguageService.cs

```diff
@@ -55,87 +63,121 @@ public override async Task<PackageInfo> GetPackageInfo(string packagePath, Cance
             PackageVersion = packageVersion,
             ServiceName = Path.GetFileName(Path.GetDirectoryName(fullPath)) ?? string.Empty,
             Language = Models.SdkLanguage.Python,
-            SamplesDirectory = Path.Combine(fullPath, "samples")
+            SamplesDirectory = Path.Combine(fullPath, "samples"),
+            SdkType = sdkType
         };
         
-        logger.LogDebug("Resolved Python package: {packageName} v{packageVersion} at {relativePath}", 
-            packageName ?? "(unknown)", packageVersion ?? "(unknown)", relativePath);
+        logger.LogDebug("Resolved Python package: {sdkType} {packageName} v{packageVersion} at {relativePath}", 
+            sdkType, packageName ?? "(unknown)", packageVersion ?? "(unknown)", relativePath);
         
         return model;
     }
 
-    private async Task<(string? Name, string? Version)> TryGetPackageInfoAsync(string packagePath, CancellationToken ct)
-    {
-        string? packageName = null;
-        string? packageVersion = null;
+private async Task<(string? Name, string? Version)> TryGetPackageInfoAsync(string packagePath, CancellationToken ct)
+{
+    string? packageName = null;
+    string? packageVersion = null;
 
-        var sdkPackagingPath = Path.Combine(packagePath, "sdk_packaging.toml");
-        if (File.Exists(sdkPackagingPath))
+    var pyprojectPath = Path.Combine(packagePath, "pyproject.toml");
```

Comment:
Do we want to call out to the python script to get these properties? WE currently use https://github.com/Azure/azure-sdk-for-python/blob/main/eng/scripts/get_package_properties.py in https://github.com/Azure/azure-sdk-for-python/blob/main/eng/scripts/Language-Settings.ps1#L176C50-L176C75